tests/test_dataset.py

```diff
@@ -25,7 +25,142 @@
+'@type':
+- prov:Entity
+- schema:Dataset
+- wfprov:Artifact
+_id: a4034ad6-dcf5-4e84-be4f-1a4a1440c800
+_label: a4034ad6-dcf5-4e84-be4f-1a4a1440c800
+_project:
+  '@context':
+    _id: '@id'
+    created: schema:dateCreated
+    creator: schema:creator
+    name: schema:name
+    prov: http://www.w3.org/ns/prov#
+    schema: http://schema.org/
+    updated: schema:dateUpdated
+    version: schema:schemaVersion
+  '@type':
+  - prov:Location
+  - schema:Project
+  _id: https://renku.ch/example/example-project
+  created: '2019-09-27T12:48:48.436000+00:00'
+  creator:
+    '@context':
+      _id: '@id'
+      affiliation: schema:affiliation
+      alternate_name: schema:alternateName
+      email: schema:email
+      name: schema:name
+      schema: http://schema.org/
+    '@type': schema:Person
+    _id: mailto:johndoe12345@example.com
+    affiliation: null
+    alternate_name: null
+    email: johndoe12345@example.com
+    name: John Doe
+  name: test project
+  updated: '2019-09-27T12:48:48.436000+00:00'
+  version: '1'
+created: '2019-09-30T09:51:21.709638+00:00'
+creator:
+- '@type': schema:Person
+  _id: mailto:johndoe12345@example.com
+  affiliation: null
+  alternate_name: null
+  email: johndoe12345@example.com
+  name: John Doe
+date_published: null
+description: ''
+files:
+- '@type':
+  - prov:Entity
+  - schema:DigitalDocument
+  - wfprov:Artifact
+  _id: blob/bd44bbfc5aa49ad98ebdc3f401533fddfda33a08/testfile
+  _label: testfile@bd44bbfc5aa49ad98ebdc3f401533fddfda33a08
+  _project:
+    '@context':
+      _id: '@id'
+      created: schema:dateCreated
+      creator: schema:creator
+      name: schema:name
+      prov: http://www.w3.org/ns/prov#
+      schema: http://schema.org/
+      updated: schema:dateUpdated
+      version: schema:schemaVersion
+    '@type':
+    - prov:Location
+    - schema:Project
+    _id: https://renku.ch/example/example-project
+    created: '2019-09-27T12:48:48.436000+00:00'
+    creator:
+      '@context':
+        _id: '@id'
+        affiliation: schema:affiliation
+        alternate_name: schema:alternateName
+        email: schema:email
+        name: schema:name
+        schema: http://schema.org/
+      '@type': schema:Person
+      _id: mailto:johndoe12345@example.com
+      affiliation: null
+      alternate_name: null
+      email: johndoe12345@example.com
+      name: John Doe
+    name: Last dataset test project
+    updated: '2019-09-27T12:48:48.436000+00:00'
+    version: '1'
+  added: '2019-09-30T11:03:09.664225+00:00'
+  creator:
+  - '@type': schema:Person
+    _id: mailto:johndoe12345@example.com
+    affiliation: null
+    alternate_name: null
+    email: johndoe12345@example.com
+    name: John Doe
+  dataset: newdata
+  name: testfile
+  path: testfile
+  url: testfile
+identifier: a4034ad6-dcf5-4e84-be4f-1a4a1440c800
+in_language: null
+keywords: []
+license: null
+name: newdata
+path: .renku/datasets/a4034ad6-dcf5-4e84-be4f-1a4a1440c800
+tags: []
+url: null
+version: null
+"""
```

Comment:
I'd rather put this in a separate file.

Reply:
We could create it as a fixture and put it into the fixture directory. If we decide to go this path I propose we create a list of all such testing fixtures (I remember there are project ones as well) and knock them out there.

Reply:
I actually decided to just generate the metadata - since the metadata is currently in flux I think this makes sense for the time being.